styles/stylesheet.css

```diff
@@ -0,0 +1,144 @@
+body {
+  background-color: rgb(236,236,236);
+  align-items: center;
+  margin: 1rem;
+}
+
+.dogface {
+  height: 280px;
+  width: 100%;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: absolute;
+  border-color: rgb(29, 71, 152);
+}
+.header {
+  position: relative;
+  margin: 5px;
+  width: auto;
+}
+.logo {
+  position: relative;
+  margin: 5px;
+  width: 33%;
+}
+.buttons {
+  font-family: inherit;
+  font-size: 100%;
+  font-color: rgb(9,60,96);
+  text-align: center;
+  box-sizing: 100%;
+  border-style: solid;
+  border-color: rgb( 236,236,236);
+  background-color: rgb( 236,236,236);
+  display: flex;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  color: rgb(236, 236,236);
+  width: 100%;
+  position: absolute;
+  margin-top: 45px;
+  }
+
+.red-shirt {
```

Comment:
not a fan of floats for this use. Maybe use grid or flexbox on the wrapping element?